package/gluon.mk

```
@@ -24,7 +24,7 @@ shell-verbatim = $(call shell-unescape,$(call shell-escape,$(1)))
 
 define GluonCheckSite
 [ -z "$$STAGING_DIR_HOSTPKG" ] || PATH="$$STAGING_DIR_HOSTPKG/bin:$$PATH"
-lua "$$IPKG_INSTROOT/lib/gluon/check-site.lua" <<'END__GLUON__CHECK__SITE'
+LUA_PATH="$$IPKG_INSTROOT/usr/lib/lua" lua "$$IPKG_INSTROOT/lib/gluon/check-site.lua" <<'END__GLUON__CHECK__SITE'
```

```suggestion
LUA_PATH="$$IPKG_INSTROOT/usr/lib/lua/?.lua" lua "$$IPKG_INSTROOT/lib/gluon/check-site.lua" <<'END__GLUON__CHECK__SITE'
```

Did not test this, but I guess that this would work here
https://www.lua.org/pil/8.1.html

At least according to the error message and the docs

@lemoer